.circleci/config.yml

```diff
@@ -29,10 +29,10 @@ jobs:
       - run:
           name: run gradle
           command: |
-            LATEST_VERSION=$(curl --silent "https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
-            [  -z "$LATEST_VERSION" ] && LATEST_VERSION="0.0.0"
-            VERSION=$(echo $LATEST_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g')
-            sed -i s/version=.*/version=$VERSION/ gradle.properties
+            # LATEST_VERSION=$(curl --silent "https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+            # [  -z "$LATEST_VERSION" ] && LATEST_VERSION="0.0.0"
+            # VERSION=$(echo $LATEST_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g')
+            # sed -i s/version=.*/version=$VERSION/ gradle.properties
```

Comment:
Why have we done this? Should either be deleted or uncommented.

Reply:
Thanks Steve , uncommented now